Lombiq.NodeJs.Extensions/package.json

```diff
@@ -16,22 +16,22 @@
   },
   "scripts": {
     "echo-env": "npm run env",
-    "build:styles": "SCSS_SOURCE=Assets/Styles SCSS_TARGET=wwwroot/css pnpm _build:styles",
-    "build:styles:args": "SCSS_SOURCE=$npm_config_source SCSS_TARGET=$npm_config_target pnpm _build:styles",
-    "_build:styles": "run-p _lint:styles _compile:styles",
+    "build:styles": "SCSS_SOURCE=Assets/Styles SCSS_TARGET=wwwroot/css pnpm _build:styles --silent",
+    "build:styles:args": "SCSS_SOURCE=$npm_config_source SCSS_TARGET=$npm_config_target pnpm _build:styles --silent",
+    "_build:styles": "run-p -c -s _lint:styles _compile:styles",
```

Comment:
Are there long switch names available, like `--silent`? User those instead, so it's easier to understand what's happening.

Reply:
Yes, there are, They just make the command so much longer. But in principle, I agree with you.